spec/actions_specs/setup_jenkins_spec.rb

```diff
@@ -0,0 +1,194 @@
+describe Fastlane do
+  describe Fastlane::FastFile do
+    describe "Setup Jenkins Integration" do
+      before :each do
+        %w(BACKUP_XCARCHIVE_DESTINATION CODE_SIGNING_IDENTITY DERIVED_DATA_PATH FL_SETUP_JENKINS_DERIVED_DATA_PATH
+           FL_SETUP_JENKINS_FORCE FL_SETUP_JENKINS_OUTPUT_DIRECTORY FL_SETUP_JENKINS_RESULT_BUNDLE
+           FL_SETUP_JENKINS_SET_CODE_SIGNING_IDENTITY FL_SETUP_JENKINS_UNLOCK_KEYCHAIN GYM_CODE_SIGNING_IDENTITY
+           GYM_DERIVED_DATA_PATH GYM_OUTPUT_DIRECTORY GYM_RESULT_BUNDLE KEYCHAIN_PASSWORD KEYCHAIN_PATH
+           SCAN_DERIVED_DATA_PATH SCAN_OUTPUT_DIRECTORY SCAN_RESULT_BUNDLE XCODE_DERIVED_DATA_PATH).each do |key|
+          ENV.delete(key)
+        end
+      end
+
+      it "doesn't work outside CI" do
```

Comment:
But `travis` runs the tests of `fastlane` too, how does that work? Maybe the `Helper.test?` should be mocked?